tests/test_knn.py

```diff
@@ -0,0 +1,46 @@
+"""Unit tests for the KNN classifier."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.euclidean import EuclideanMetric
+from geomstats.learning.knn import KNearestNeighborsClassifier
+
+TOLERANCE = 1e-4
+
+
+class TestOnlineKmeansMethods(geomstats.tests.TestCase):
+    """Class defining the Poincare polydisk tests."""
+
+    def setUp(self):
+        """Define the parameters to test."""
+        gs.random.seed(1234)
+        self.n_neighbors = 3
+        self.metric = EuclideanMetric
```

Comment:
Should be an object, see other unit test files for example

Reply:
TODO (maybe next PR): test if the method works with a non-Euclidean metric?

Reply:
For the example of the next PR, I plot the result of a classification on the sphere, so it works for other metrics.